utils/aeternity.js

```diff
@@ -57,59 +53,42 @@ class Aeternity {
         .filter(tx => tx.tx.type === 'ContractCallTx'));
   }
 
-  async transactionEvents(hash) {
-    const fetchTransactionEvents = async () => {
-      const tx = await this.client.tx(hash);
-      const microBlock = await this.client.getMicroBlockHeader(tx.blockHash);
-
-      const eventsSchema = [
-        { name: 'TipReceived', types: [SOPHIA_TYPES.address, SOPHIA_TYPES.int, SOPHIA_TYPES.string] },
-        { name: 'ReTipReceived', types: [SOPHIA_TYPES.address, SOPHIA_TYPES.int, SOPHIA_TYPES.string] },
-        { name: 'TipWithdrawn', types: [SOPHIA_TYPES.address, SOPHIA_TYPES.int, SOPHIA_TYPES.string] },
-        { name: 'QueryOracle', types: [SOPHIA_TYPES.string, SOPHIA_TYPES.address] },
-        { name: 'CheckPersistClaim', types: [SOPHIA_TYPES.string, SOPHIA_TYPES.address, SOPHIA_TYPES.int] },
-      ];
-
-      const decodedEvents = decodeEvents(tx.log, { schema: eventsSchema });
-
-      return decodedEvents.map(decodedEvent => ({
-        event: decodedEvent.name,
-        address: `ak_${decodedEvent.decoded[1]}`,
-        amount: decodedEvent.decoded[2] ? decodedEvent.decoded[2] : null,
-        url: decodedEvent.decoded[0],
-        caller: tx.tx.callerId,
-        nonce: tx.tx.nonce,
-        height: tx.height,
-        hash: tx.hash,
-        time: microBlock.time,
-      }));
-    };
-
-    return this.cache
-      ? this.cache.getOrSet(['transactionEvents', hash], () => fetchTransactionEvents())
-      : fetchTransactionEvents();
-  }
+  async fetchTransactionEvents(hash) {
+    const tx = await this.client.tx(hash);
```

Comment:
why remove the cache here? it's still needed for now, otherwise the request on mainnet will always time out, cache takes ~15min to build

Reply:
the cache is moved out of aeternity js and in the place where we call "fetchTransactionEvents" in the CacheLogic

Reply:
okay, I see, that is fine